src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.component.ts

```diff
@@ -10,16 +10,18 @@ import { Observable } from 'rxjs/Observable';
     templateUrl: './dot-edit-page-workflows-actions.component.html',
     styleUrls: ['./dot-edit-page-workflows-actions.component.scss']
 })
-export class DotEditPageWorkflowsActionsComponent implements OnInit {
-    @Input() inode: string;
+export class DotEditPageWorkflowsActionsComponent implements OnChanges {
+    @Input() pageState: any;
     @Input() label: string;
 
+    inode: string;
```

Comment:
I think we don't need this attribute because we can use 'this.pageState.workingInode' directly

Reply:
I'll prefer to leave it, to keep just `inode` more readable instead of a large variable name